File: python/database_test.py
#!/usr/bin/python
# -*- coding: utf-8 -*-

import sqlite3 as lite
import sys

con = None

try:
    con = lite.connect('../sql/bookmarks.db')
    
    cur = con.cursor()    
    cur.execute('SELECT * FROM bookmarks')
    
    rows = cur.fetchall()

    for row in rows:
        print row[1]
        print row[2]
    
except lite.Error, e:
    
    print "Error %s:" % e.args[0]
    sys.exit(1)
    
finally:
    
    if con:
        con.close()

File: nodejs/database_test.js
var sqlite3 = require('sqlite3').verbose();  
var file = "../sql/bookmarks.db";  
var db = new sqlite3.Database(file);  
db.all("SELECT * FROM bookmarks", function(err, rows) {  
        rows.forEach(function (row) {  
            console.log(row.title, row.url);  
        })  
    });   
db.close(); 